### PR TITLE
updated My Stakes page for multi-chain

### PR DESCRIPTION
**Pull request by Israellund**
## Link to Issue
Closes: #7414 

## Description of Changes
-Added new column for Chain on both `My stake` and `Transaction history` tabs/pages
-changed BASE to ETH denomination
-refactored code

## Test Plan
-go to My Community Stake page 
-confirm you now see the new Chain column in `My stake`, as well as the ETH denomination in Avg. price
-click `Transaction history` and confirm you now see the Chain column
Screenshot 2024-04-16 at 11 01 06 AM
Screenshot 2024-04-16 at 11 01 19 AM

### CODE REVIEW

**Review by burtonator — APPROVED** [review]

**Review by ForestMars — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Stakes/Stakes.tsx [hunk 1]
```diff
@@ -87,10 +92,7 @@ const Stakes = ({ transactions }: TransactionsProps) => {
       .map((transaction: any) => ({
         ...transaction,
         voteWeight: transaction.voteWeight + 1, // total vote weight is +1 of the stake weight
-        avgPrice: `${transaction.avgPrice.toFixed(5)} ${getCommunityStakeSymbol(
-          app.config.chains.getById(transaction?.community?.id)?.ChainNode
-            ?.name || '',
-        )}`,
+        avgPrice: `${transaction.avgPrice.toFixed(5)} ${'ETH'}`,
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Stakes/Stakes.tsx:95]: i think this should be dynamic per, similar to what we are doing in https://github.com/hicommonwealth/commonwealth/pull/7205

Comment (Israellund) [reply packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Stakes/Stakes.tsx:95]: @mzparacha I agree, let me change it. I didn't like this solution so thank you for the suggestion.

Comment (Israellund) [reply packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Stakes/Stakes.tsx:95]: @mzparacha After speaking with Zak Hap, as of now at no point will either BASE or Blast not be ETH, therefore keeping `ETH` hardcoded for now is fine. When we expand and allow other ways to buy stake that will require a bigger build out and refactoring of this line of code.

Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Stakes/Stakes.tsx:95]: i see, i think we can resolve this comment then. Thanks

**Pull request merged**